### PR TITLE
Create john-hutchins.md

### PR DESCRIPTION
**Pull request by ovesyan**

### CODE REVIEW

File: history/john-hutchins.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+description: English historian of machine translation
+---
```
Comment (bittlingmayer) [inline history/john-hutchins.md:3]: Title is missing

File: history/john-hutchins.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+description: English historian of machine translation
```
Comment (bittlingmayer) [inline history/john-hutchins.md:2]: No nationality

> Linguist and early machine translation researcher

**Review by bittlingmayer — APPROVED** [review]

**Comment by ovesyan** [comment]
Adam, could you edit the article and delete the space between the square brackets and the parentheses?

**Comment by bittlingmayer** [comment]
@ovesyan Done

**Pull request merged**